bids2nda/main.py

```diff
@@ -162,11 +162,22 @@ def run(args):
         dict_append(image03_dict, 'image_modality', "MRI")
         dict_append(image03_dict, 'scanner_manufacturer_pd', metadata.get("Manufacturer", ""))
         dict_append(image03_dict, 'scanner_type_pd', metadata.get("ManufacturersModelName", ""))
-        dict_append(image03_dict, 'scanner_software_versions_pd', metadata.get("HardcopyDeviceSoftwareVersion", ""))
+        dict_append(image03_dict, 'scanner_software_versions_pd', metadata.get("SoftwareVersions", ""))
```

Comment:
+1 -- I do not see the HardcopyDeviceSoftwareVersion present anywhere.
If they were interested in all "versions", then we could also extend it with ConversionSoftwareVersion but since they do not care -- "better" for us, worse for reproducibility ;)

Reply:
I think this one is settled